.travis.yml

```diff
@@ -2,6 +2,5 @@ language: node_js
 node_js:
   - "6"
   - "8"
-  - "9"
```

Comment:
I'm ok with this - 9.x was never LTS and was EOL 2018-06-30

Reply:
Okay.